### PR TITLE
chore: updated links + public stats in LinuxGamer.json

### PR DESCRIPTION
**Pull request by LinuxGamer**
## Changes proposed
Adds Mastodon link and profile view count

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2298"></a>

### CODE REVIEW

File: data/LinuxGamer.json [hunk 1]
```diff
@@ -19,6 +20,11 @@
       "url": "https://twitter.com/the_linuxgamer",
       "icon": "twitter"
     },
+    {
+      "name": "Follow my Mastodon",
+      "URL": "https://fosstodon.org/@linuxgamer",
+      "icon": "mastodon"
+     },
```
Comment (schmelto) [inline data/LinuxGamer.json:27]:
```suggestion
    },
```

Comment (LinuxGamer) [reply data/LinuxGamer.json]: Fixed this now, thanks for pointing it out, couldn't see what the check didn't like XD

**Comment by LinuxGamer** [comment]
I have no idea what the issue is now, anyone see what's wrong?

File: data/LinuxGamer.json [hunk 2]
```diff
@@ -58,4 +64,4 @@
       "description": "Pledged to become Greener and more sustainable within development and content creation."
     }
   ]
-}
+}
```
Comment (Pradumnasaraf) [inline data/LinuxGamer.json:67]:
```suggestion
}

```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, an ending line is required, otherwise looks good!

File: data/LinuxGamer.json (hunk 2 above)
Comment (LinuxGamer) [reply data/LinuxGamer.json]: fixed now :) thanks

Comment (LinuxGamer) [reply data/LinuxGamer.json]: fixed now :) thanks

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**